### PR TITLE
OPENEUROPA-3006: Add support for event listing to patterns

### PR DESCRIPTION
**Pull request by ademarco**

### CODE REVIEW

**Review by nagyad — DISMISSED** [review]

File: tests/Kernel/fixtures/rendering.yml [hunk 1]
```diff
@@ -1717,6 +1759,34 @@
...
+- array:
+    '#type': pattern
+    '#id': featured_item
+    '#variant': "default"
+    '#fields':
+      title: "Example title"
+      description: "Example description"
+  assertions:
+    count:
+      "footer.ecl-card__footer": 0
```
Comment (imanoleguskiza) [inline tests/Kernel/fixtures/rendering.yml]: We could just probably add this assertion to one of the previous tests and save creating one array just for this.

File: templates/patterns/list_item/list_item.ui_patterns.yml [hunk 2]
```diff
@@ -23,6 +20,18 @@ list_item:
     thumbnail_secondary:
       label: "List item: thumbnail with secondary image"
       description: "Thumbnail list item, contains a secondary image, metadata information and a linked title."
+    date:
+      label: "List item: event"
```
Comment (imanoleguskiza) [inline templates/patterns/list_item/list_item.ui_patterns.yml:24]: Its weird to call this pattern "event" since it doesn't necessarily involve an event.

**Review by imanoleguskiza — CHANGES_REQUESTED** [review]

File: templates/patterns/list_item/list_item.ui_patterns.yml (hunk 2 above)
Comment (nagyad) [reply templates/patterns/list_item/list_item.ui_patterns.yml:24]: Probably it's because it is used on an event listing item at the moment.

Comment (ademarco) [reply templates/patterns/list_item/list_item.ui_patterns.yml:24]: You are right @imanoleguskiza I'll change their name to past date, upcoming date etc.

File: tests/Kernel/fixtures/rendering.yml (hunk 1 above)
Comment (ademarco) [reply tests/Kernel/fixtures/rendering.yml]: Ok I'll move it up.

File: templates/patterns/list_item/list_item.ui_patterns.yml (hunk 2 above)
Comment (ademarco) [reply templates/patterns/list_item/list_item.ui_patterns.yml:24]: Still ongoing date doesn't make much sense :D

Comment (imanoleguskiza) [reply templates/patterns/list_item/list_item.ui_patterns.yml:24]: Actually you are right, the date blocks are just for events: https://ec.europa.eu/component-library/ec/components/date-block/usage/

Comment (ademarco) [reply templates/patterns/list_item/list_item.ui_patterns.yml:24]: We agreed with @imanoleguskiza to keep it event. resolving.

**Review by imanoleguskiza — APPROVED** [review]

**Review by nagyad — APPROVED** [review]

**Pull request merged**